extensions/websockets-next/deployment/src/main/java/io/quarkus/websockets/next/deployment/WebSocketProcessor.java

```diff
@@ -448,7 +448,9 @@ void validateConnectorInjectionPoints(List<WebSocketEndpointBuildItem> endpoints
                         .noneMatch(clientEndpointType.name()::equals)) {
                     validationErrors.produce(
                             new ValidationErrorBuildItem(new WebSocketClientException(String.format(
-                                    "Type argument [%s] of the injected WebSocketConnector is not a @WebSocketClient endpoint: %s",
+                                    "Type argument [%s] of the injected WebSocketConnector is not a @WebSocketClient endpoint, because it is not a bean: %s"
```

Comment:
In theory, you can do `@Inject WebSocketConnector<NotAClientEndpoint>` where `NotAClientEndpoint` is not annotated with `@WebSocketClient` but it can have a scope and therefore it's a CDI bean. In this case, the new messages would be even more confusing...